apps/frontend/src/views/Login.vue

```diff
@@ -71,31 +136,47 @@ import Vue from 'vue';
 import Component from 'vue-class-component';
 import UserValidatorMixin from '@/mixins/UserValidatorMixin';
 import {required, email} from 'vuelidate/lib/validators';
+import {LocalStorageVal} from '@/utilities/helper_util';
 import {ServerModule} from '@/store/server';
 import {SnackbarModule} from '@/store/snackbar';
 
+const login_tab = new LocalStorageVal<string>('login_curr_tab');
+
 export interface LoginHash {
   email: string;
   password: string;
 }
 
+export interface LDAPLoginHash {
+  username: string;
+  password: string;
+}
+
 @Component({
   mixins: [UserValidatorMixin],
   validations: {
     email: {
       required,
       email
     },
+    username: {
+      required
+    },
     password: {
       required
     }
   }
 })
 export default class Login extends Vue {
   email: string = '';
+  username: string = '';
   password: string = '';
+  active_tab: string = login_tab.get_default('logintab-standard')
 
   mounted() {
+    if(!ServerModule.ldap){
+      document.querySelectorAll('[role="tablist"]')[0].remove()
+    }
```

Comment:
Modifying the DOM directly like this seems like bad practice. Can you just add a `v-if` to the `v-tab` above to hide the LDAP tab if LDAP is not available?

Reply:
Unfortunately the tablist component isn't accessible, and only hiding v-tab still shows the container it sits on

Reply:
Adding `:height="ldapenabled ? '' : 0"` to the `v-show` seems to have the equivalent behavior without modifying the DOM.

Reply:
Changed to just always show "Heimdall Login" and "Organization Login" if supported.